From 0bb3742f51d1af0ebc1394c72e899524b0adf81d Mon Sep 17 00:00:00 2001
From: Martin Weinelt <martin@darmstadt.freifunk.net>
Date: Wed, 14 Nov 2018 17:28:56 +0100
Subject: [PATCH] modules: update routing packages

bc6e7f6 Merge pull request #425 from ecsv/batadv-18.06
c00a1bf alfred: Merge bugfixes from 2018.4
7bf62cc batman-adv: Merge bugfixes from 2018.4
d1cf6d1 Merge pull request #422 from ecsv/batadv-18.06
121c92d batctl: Don't overwrite default PKG_BUILD_DIR
95e56cf alfred: Add openwrt revision to internal version
93cce26 batctl: Add openwrt revision to internal version
1a83b56 batman-adv: Add openwrt revision to internal version
1ba424a batman-adv: Load module with AutoProbe
b9656fb batman-adv: Fix search for cfg80211 module symbols
059d726 batman-adv: Remove unused option "interfaces" in config
3d5c2d0 batman-adv: Add DEBUG_LOG -> DEBUGFS dependency
a9a4b04 batman-adv: Rebuild automatically on mac80211 changes
---
 modules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules b/modules
index b0169da..266d9e2 100644
--- a/modules
+++ b/modules
@@ -13,7 +13,7 @@ PACKAGES_GLUON_COMMIT=be2c35785994e443d895225c7240474a46f64f5e
 
 PACKAGES_ROUTING_REPO=https://github.com/openwrt-routing/packages.git
 PACKAGES_ROUTING_BRANCH=openwrt-18.06
-PACKAGES_ROUTING_COMMIT=8b760071d17ca1245726903aae0d05d5ea6fe6e6
+PACKAGES_ROUTING_COMMIT=bc6e7f6903c8237c77131aedfc92dba40e1bc6ac
 
 PACKAGES_LUCI_REPO=https://github.com/openwrt/luci.git
 PACKAGES_LUCI_BRANCH=lede-17.01
-- 
2.5.0

